pkg/dataobj/index/indexobj/builder.go

```diff
@@ -374,6 +404,13 @@ func (b *Builder) observeObject(ctx context.Context, obj *dataobj.Object) error
 
 	for _, sec := range obj.Sections() {
 		switch {
+		case indexpointers.CheckSection(sec):
+			indexPointerSection, err := indexpointers.Open(context.Background(), sec)
```

Comment:
Interesting, I just noticed we have a context available yet we use context.Background() for all of these. We should use the ctx param